Fix bus

tomkenmag opened the pull request:
Reimplemented the bus endpoint using the new api. Old tests and old documentation used.

### Checklist

- [X] Write tests
- [X] Write documentation

MiniGod commented on endpoints/bus/realtime.js:
```diff
@@ -3,12 +3,26 @@
+    sudurnesBusRoutes,
+    reykjanesBusRoutes)
+
+  request.get('https://app.straeto.is/pele/api/v1/positions/filter/' + allBusRoutes.join('%2C'), function (error, response, body) {
```
I see no reason for this request, but since the old implementation did it as well, I don't see this as a stopper. We can improve on that later.

tomkenmag replied:
Agree, also the new endpoint can handle invalid route numbers so there's no point in doing this check.

MiniGod commented on endpoints/bus/realtime.js:
```diff
@@ -56,36 +80,29 @@ const getBusRoutes = (data) => new Promise((resolve, reject) => {
+              'x': bus.lat,
+              'y': bus.lon,
+              'from': bus.lastStop,
+              'to': bus.nextStop
```
`from` and `to` used to be the names of the stops, but they are now some ID numbers. e.g. `90000635` and `90000366`. That's a breaking change, but better than nothing...

tomkenmag replied:
This can be fixed, the table containing all the bus stops is available.

MiniGod replied:
That would actually be great! Whether it is hard-coded or another call to the straeto API, I don't really care.

tomkenmag replied:
I'll patch it up after next week.

tomkenmag commented:
I just rewrote this a bit and now we have a huge file containing all the bus stops in Iceland managed by Strætó, but at least we get the names of the bus stops. And also, the timestamps weren't being parsed at all so that's also fixed.

kristjanmik commented:
Can this be merged?
